build/ios.js

```diff
@@ -80,7 +54,6 @@ IOS.prototype.package = function (packager, next) {
 	const DEST_IOS = path.join(packager.zipSDKDir, 'iphone');
 
 	async.parallel([
-		this.fetchLibTiCore.bind(this),
```

Comment:
Also need to remove the step below where we try to copy iphone/headers/JavaScriptCore since there's no files there anymore (that's why the build is failing right now, lines 62-64 in your modified file) - and the 'headers' entry in the array of folders/files we try to copy (line 66 in your modified file)